nautilus_trader/adapters/dydx/execution.py

```diff
@@ -1027,7 +1027,7 @@ async def _submit_order_single(self, order) -> None:
             )
             return
 
-        if dydx_order_tags.is_short_term_order:
+        if dydx_order_tags.is_short_term_order and not dydx_order_tags.is_conditional_order:
```

Comment:
Wondering why an additional order tag is required. Can we also use the order type directly?

Reply:
we can create a condition with order type. but as the structure followed in the code with the order tag we used the same if required we can change it .
please let us know what you suggest.